### PR TITLE
[fix bug 1273837] Decommission Hello FTU and redirect to Hello product page

### PR DESCRIPTION
**Pull request by alexgibson**
## Description
- Removes the template pages found at `/firefox/%VERSION%/hello/start` and redirects to `firefox/hello`.

## Bugzilla link
https://bugzilla.mozilla.org/show_bug.cgi?id=1273837

## Testing
I added tests for the redirect, although an extra pair of eyes on the redirect regexp would be appreciated.

## Checklist
- [x] Requires l10n changes.
- [x] Related functional & integration tests passing.

### CODE REVIEW

**Comment by alexgibson** [comment]
Ping @flodolo - there may be some lang files we can remove here too.

**Comment by flodolo** [comment]
.lang files no, strings definitely.

File: bedrock/firefox/redirects.py [hunk 1]
```diff
@@ -498,4 +498,7 @@ def firefox_mobile_faq(request, *args, **kwargs):
 
     # bug 1260423
     redirect(r'^firefox/choose/?$', 'firefox.new'),
+
+    # bug 1273837
+    redirect(r'^firefox(?:\/\d+\.\d+\.?(?:\d+)?\.?(?:\d+)?(?:a?)(?:\d*))?/hello/start/?$', 'firefox.hello'),
```
Comment (jpetto) [inline bedrock/firefox/redirects.py]: I think there are one too many digit capturing groups here - the above would match `46.0.1.2` and `46.0.1.2a2`, which I don't think we need (uh, right?).

Assuming an `a` must always be followed by a number, I think you simplify slightly to:

`^firefox(?:\/\d+\.\d+(?:\.\d+)?(?:a\d+)?)?/hello/start/?$`

So, "firefox" followed by at least `#.#`, then optionally another `.#` and optionally `a#` at the end.

**Comment by jpetto** [comment]
Can also get rid of the bundles in `static_media.py`?

**Comment by jpetto** [comment]
Small suggestion on the regex and note to clean up `static_media.py`, but otherwise looks good. 🚎 r+wc

File: bedrock/firefox/redirects.py (hunk 1 above)
Comment (alexgibson) [inline bedrock/firefox/redirects.py]: Good catch on the extra digit, will update 👍

**Pull request merged**